openlibrary/core/civicrm.py

```diff
@@ -34,7 +34,7 @@ def get_contact(username=None, contact_id=None):
             params=data,
             timeout=3,
             headers=dict(
-                Authorization=f"Basic {lending.config_ia_civicrm_api.get('auth', '')}"
+                Authorization=f"basic {lending.config_ia_civicrm_api.get('auth', '')}"
```

Comment:
Lowercase keys.  Do not change the values.
```suggestion
                Authorization=f"Basic {lending.config_ia_civicrm_api.get('auth', '')}"
```